resources/zib/zib-TreatmentObjective.xml

```diff
@@ -0,0 +1,167 @@
+  <description value="The treatment objective describes the desired result of the treatment/interventions in terms of the patient’s diagnosed problem. The objective and target date are entered in addition to the target value. Comparing the healthcare results to the treatment objective provides insight into the effectivity of the treatment/interventions." />
+  <purpose value="This Goal resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) TreatmentObjective v3.2 (2020)](https://zibs.nl/wiki/TreatmentObjective-v3.2(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-treatmentobjective-v3.2-2020EN" />
+    <uri value="https://zibs.nl/wiki/TreatmentObjective-v3.2(2020EN)" />
+    <name value="zib TreatmentObjective-v3.2(2020EN)" />
+  </mapping>
+  <mapping>
+    <identity value="zib-functionalormentalstatus-v3.2-2020EN" />
+    <uri value="https://zibs.nl/wiki/FunctionalOrMentalStatus-v3.2(2020EN)" />
+    <name value="zib FunctionalOrMentalStatus-v3.2(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Goal" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Goal" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Goal">
+      <path value="Goal" />
+      <short value="TreatmentObjective" />
+      <comment value="The concepts of FunctionalOrMentalStatus are individually modulated on the elements of Goal's FHIR resource. A direct reference was not possible on the .target element." />
+      <alias value="Behandeldoel" />
+      <mapping>
+        <identity value="zib-treatmentobjective-v3.2-2020EN" />
+        <map value="NL-CM:13.5.1" />
+        <comment value="TreatmentObjective" />
+      </mapping>
+    </element>
+    <element id="Goal.description.text">
+      <path value="Goal.description.text" />
+      <short value="DesiredHealthcareResult" />
+      <definition value="The textual account of the treatment objective. This concept provides the option to define a treatment objective which cannot be described as a measurement value or functional/mental status." />
+      <alias value="GewenstZorgresultaat" />
+      <constraint>
+        <key value="zib-TreatmentObjective-1" />
+        <severity value="error" />
+        <human value="One concept from DesiredHealthCondition::FunctionalOrMentalStatus and DesiredHealthcareResult must be selected." />
+        <source value="http://nictiz.nl/fhir/StructureDefinition/zib-TreatmentObjective" />
+      </constraint>
```

Comment:
Heeft geen `constraint.expression` dus doet niks.

Heb je https://simplifier.net/guide/profilingacademy/Customconstraints doorgenomen? En naar zib-Alert gekeken wat er daar gebeurd (#129 )? Wat ik daar ook heb gedaan is wat handmatige voorbeelden gemaakt (een invalid en twee goede) die de constraint daadwerkelijk test in de validator.

Ik vind het lastig om feedback te geven terwijl er niets staat om tips op te geven.

Reply:
See #125

Reply:
Dit is nu volgens mij opgelost en beschreven in `.description.text`.